scrap.js

```diff
@@ -17,20 +17,78 @@ async function fetchLeaders(id) {
   return leaders
 }
 
-async function fetchOrgsWithLeaders() {
+async function searchGitHubOrgs(query) {
+  const token = process.env.GITHUB_TOKEN
+  const res = await fetch(
+    `https://api.github.com/search/users?q=${query}%20type:org`,
+    {
+      headers: token ? { Authorization: `token ${token}` } : {}
+    }
+  )
+  const { items } = await res.json()
+  return items || []
+}
+
+async function findOrganization({
+  name,
+  description,
+  mailing_list,
+  website_url,
+  irc_channel,
+  blog_url,
+  guide_to_working_url,
+}) {
+  const githubPattern = /(?:https?:\/\/)?(?:github\.com|gitter\.im)\/([a-zA-Z0-9-]+)/i
```

Comment:
This will be a long line that another task is going to have to fix.
And you will loose ownership (blame) of these lines when they fix it.